Yoast/Yoast-theme-public
# Changes to plugin pages, buy button and sticky bottom.

Goes along with: https://github.com/Yoast/yoast.com-ms/pull/171

**How to test:**

1. Go to http://yoast.dev/wordpress/plugins/seo/
2. Copy the current Live page to your local.
3. Select the new CMS field "Full width content (100%)"
3. Give the comparison tables the class "compare-free-plugin" and delete the inline styling on the table element.
4. Place a table around the USP's and image (Table, Tr, Td) and give the table the classes "compare-free-premium usps-paymentmethods".
5.  Edit changes / check changes

**What changed**
- New CMS field "Sentence below USPs" that gives a sentence below the USP's on top.
- New CMS checkbox "Full width content (100%)". This gives an extra class to the content section for sytling
- All plugin buttons have the copy "get" in front, and "now" behind the button name.
- In the sticky bottom there is a link to the free download. Also the microcopy changed behind the button
- All kind of styling to fix the pages.

Overal, the page should look like if you change all things:
![product](https://user-images.githubusercontent.com/6122172/30120149-59b105e6-9328-11e7-9f38-a5ed96fbe074.png)
